### DISCUSSION TITLE
Change URL/Public dns on docker install

### DISCUSSION THREAD

**Discussion by rbtved** [requarks/wiki#6652] [Help / Questions] [answered]
Hi,

I'm kinda new to docker and was looking for a way to change the url on my Wiki installation in docker. I was able to find a way into the bash on the docker, but the config.yml file contained refs to settings not in the documentation (that i could find), so changing it there does not seem to be the correct way. 

Can anyone provide some guidance on how to change the url the correct way?

**Comment by NGPixel** [comment]
The only place where you need to change the URL is in the admin area > general. There's nothing to change in docker or the OS.

**Reply by rbtved** [reply]
So that will refresh the SSL cert config too?

**Reply by NGPixel** [reply, marked as answer]
You didn't mention an SSL cert in your initial post. Then you would need to recreate the container with the new domain (assuming you're using let's encrypt).

**Reply by rbtved** [reply]
Okay makes sense. The only problem with that is that no matter how i try to recreate the container the HTTPS service does not seem to work so only HTTP works. 
Im using this to create the container:
 docker run -d -p 80:3000 -p 443:3443 --network wikinet -e "LETSENCRYPT_DOMAIN=mydomain" -e "LETSENCRYPT_EMAIL=mail@mail.dk" --name wiki --restart unless-stopped -e "DB_TYPE=postgres" -e "DB_HOST=db" -e "DB_PORT=5432" -e "DB_USER=wiki" -e "DB_PASS=dbpass" -e "DB_NAME=wiki" ghcr.io/requarks/wiki:2 

What am i doing wrong here?

**Reply by NGPixel** [reply]
post the logs from `docker logs wiki`

**Reply by rbtved** [reply]
I found a way of making it work, for some reason when creating the docker with the above command i was not starting the HTTPS service, nothing in the log about HTTPS only about HTTP. I then installed Portainer to manage the docker instance and used that to recreate, that for some reason worked. My knowledge of docker is to little to have any idea of why. 

Anyways thanks for your help @NGPixel, you pointed me in the right direction!
